docs/apache-airflow/concepts.rst

```diff
@@ -1137,6 +1136,30 @@ TaskGroup can be created using ``@task_group`` decorator, it takes one argument
     :end-before: [END howto_task_group_decorator]
 
 
+Edge Labels
+===========
+
+As well as grouping tasks into groups, you can also label the edges between
+different tasks in the Graph View - this can be especially useful for branching
+areas of your DAG, so you can label the conditions under which certain branches
+might run.
```

Comment:
nit: Probably a screenshot in this doc might be helpful

Reply:
Don't know how I didn't think about that one given it's an _entirely visual feature!_ I'll get it in tomorrow.

Reply:
I'm gonna merge this PR so we don't end up in CI hell, but @andrewgodwin please make another PR with this screenshot if you think that will help

Reply:
@andrewgodwin can you also add some Edge Labels to a few of the DAGs in `airflow/example_dags/` in the same PR? It would be useful for this feature to be exposed during local development.